components/displayNotification.js

```diff
@@ -0,0 +1,28 @@
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+export default function DisplayNotification(type, msg) {
+  if (type === 'Success') {
+    return toast.success(msg, {
+      position: 'top-center',
+      autoClose: 3000,
+      hideProgressBar: false,
```

Comment:
IMO, we should display the progress bar so that users know the alert will go away without the need for user interaction. However, I am okay with leaving it hidden if there is a reason to.

Reply:
The progress bar is not hidden. The value of hideProgressBar is false so it is displayed.

Reply:
> The progress bar is not hidden. The value of hideProgressBar is false so it is displayed.

Ah sorry about that! Was reviewing somewhat quickly. Thanks!